File: MovieHostingLinkGripper/src/main/java/com/sheryv/tools/movielinkgripper/search/TmdbSeason.java

package com.sheryv.tools.movielinkgripper.search;

import java.util.List;

import com.fasterxml.jackson.annotation.JsonIgnore;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonIgnoreType;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

@Data
@JsonIgnoreProperties(ignoreUnknown = true)
public class TmdbSeason {

    private String _id;
    @JsonProperty("air_date")
    private String airDate;
    private List<TmdbEpisode> episodes;

}

File: WebsiteScraper/src/main/kotlin/com/sheryv/tools/websitescraper/service/videosearch/TmdbSeason.kt
package com.sheryv.tools.websitescraper.service.videosearch

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty

@JsonIgnoreProperties(ignoreUnknown = true)
class TmdbSeason(
  val id: Long,
  val _id: String,
  val episodes: List<TmdbEpisode>,
  @JsonProperty("air_date")
  val airDate: String? = null,
) {
}
